docs/linuxInstall.md

```diff
@@ -1,5 +1,14 @@
 # Linux install instructions
 
+## Snap
+
+Install <project> in seconds on [Ubuntu and other snap supported Linux distributions](https://snapcraft.io/docs/core/install) with:
+
+    snap install brave --beta
+
+Installing a snap is very quick. Snaps are secure. They are isolated with all of their dependencies. Snaps also auto update when a new version is released.
```

Comment:
> Installing a snap is very quick. Snaps are secure.

this sound like advertising. I'd recommend to remove them unless the team would agree.